wordpress-mobile/WordPress-iOS
# Add login with Google button

Refs #7675 

Add the Google login button that will launch the new login process (which currently doesn't exist).

![simulator screen shot aug 25 2017 1 05 22 pm](https://user-images.githubusercontent.com/517257/29728731-64be80c6-8996-11e7-81ad-169d38d1edbc.png)

To test:
- Start the login process
- Make sure you see the Google button
- If you're a real keener, make sure it works in a variety of situations:



![google login button scenarios](https://user-images.githubusercontent.com/517257/29729734-9f4ad6a0-899a-11e7-81a6-12d1027c461d.png)



Needs review: @aerych
